javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>
+      Learn Basic String and Array Methods by Building a Music Player App
+    </title>
+  </head>
+  <body onload="app()">
```

Comment:
We have traditionally loaded the script directly, rather than calling an onload function.

I think we should discuss whether we want to teach this approach.

Reply:
Instead of using the onload property, we have just call the app function at the end of scripts file. But, I do think it would be good to teach this approach too.

Reply:
I am not opposed to showing this alternative onload function.
I think we should add a comment above the opening body tag mentioning this is the first time this attribute is used. 
```suggestion
<!-- this is the first time the onload attribute is used. We need to make sure to teach this new concept in the steps -->
  <body onload="app()">
```